app/bundles/CampaignBundle/Form/Type/EventCanvasSettingsType.php

```diff
@@ -25,16 +26,16 @@ class EventCanvasSettingsType extends AbstractType
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->add('droppedX', 'hidden');
+        $builder->add('droppedX', HiddenType::class);
 
-        $builder->add('droppedY', 'hidden');
+        $builder->add('droppedY', HiddenType::class);
     }
 
     /**
      * @return string
      */
-    public function getName()
+    public function getBlockPrefix()
     {
-        return 'campaignevent_canvassettings';
+        return self::class;
```

Comment:
```suggestion
        return 'campaignevent_canvassettings';
```
Keep the original string as it's used in a JS file as HTML element identifier.